src/content/en/2020/markup.md

```diff
@@ -457,22 +483,58 @@ Yet the most popular attribute is `class`—with almost 3 billion occurrences in
 </figure>
 
-The `value` attribute, which specifies the value of an `input` element, surprisingly completes the top ten—surprising because on our end, subjectively, we didn’t get the impression values were used that frequently.
+The most popular attribute is `class`, with nearly 3 billion occurrences in our dataset and constituting 34% of all attributes in use. `class` is by far the most prevalent attribute.
+
+The `value` attribute, which specifies the value of an `input` element, surprisingly completes the top ten. It's surprising to us because, subjectively, we didn't get the impression values were used that frequently.
 
 ### Attributes on pages
 
-Are there attributes that we find in every document? No—but almost. `href` (99.21%), `src` (99.18%), `content` (98.88%), `name` (98.61%), `type` (98.55%), `class` (98.24%), `rel` (97.98%), `id` (97.46%), `style` (95.95%), and `alt` (90.75%) make the Top 10 here, with each being found on at least 90% of sampled pages.
+Are there attributes that we find in every document? Not quite, but almost:
 
-This raises questions we cannot answer like yes, `type` is used on other elements, too, but why this tremendous popularity, given that it’s usually not needed to specify for style sheets or scripts (with CSS and JavaScript being assumed default)? Or how do we really fare with `alt`, do those 9.25% of pages that don’t include it not contain any images?
+<figure markdown>
+Element | Pages (%)
+-- | --
+href | 99.21%
+src | 99.18%
+content | 98.88%
+name | 98.61%
+type | 98.55%
+class | 98.24%
+rel | 97.98%
+id | 97.46%
+style | 95.95%
+alt | 90.75%
+title | 85.50%
+target | 82.98%
+async | 81.96%
+lang | 79.10%
+width | 78.99%
+height | 76.36%
+charset | 75.44%
+value | 67.57%
+role | 62.97%
+action | 62.34%
+http-equiv | 62.30%
+method | 60.27%
+media | 60.14%
+property | 55.68%
+placeholder | 54.21%
+
+<figcaption>{{ figure_link(
+  caption="Top 25 attributes by page.",
+  sheets_gid="1185369559",
+  sql_file="pages_almanac_by_device_and_attribute_name_present.sql"
+)}}</figcaption>
+</figure>
 
-To name all the attributes found on at least half of the pages, we would add `title` (85.5%), `target` (82.98%), `async` (81.96%), `lang` (79.1%), `width` (78.99%), `height` (76.36%—note the difference to `width`), `charset` (75.44%), `value` (67.57%), `role` (62.97%), `action` (62.34%), `http-equiv` (62.3%), `method` (60.27%), `media` (60.14%), `property` (55.68%), and `placeholder` (54.21%).
+These results raise some questions that we cannot answer. For example, `type` is used on other elements too, but why this tremendous popularity? Especially given that it's usually not needed to specify for style sheets or scripts, with CSS and JavaScript being assumed default. Or, how do we really fare with `alt`? Do those 9.25% of pages not contain any images or are they just inaccessible? The rest of these 25 attributes make up all of those found on at least half of all pages.
```

Comment:
This will need editing after the table got shortened. It may suffice to remove the last sentence, “The rest of these 25 attributes make up all of those found on at least half of all pages.”.